crypto/src/hash/types.rs

```diff
@@ -18,7 +18,7 @@ use wire_type::WireType;
 
 pub(crate) type HashBytes = Box<[u8]>;
 
-#[derive(Clone, Default, Debug, PartialEq, Serialize, Deserialize)]
+#[derive(Copy, Clone, Default, Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
Can we not use Copy here. Deriving Copy is implicit here. Instead we can call `clone()` where we need them, or just take ref of the BaseHash

Reply:
We have to use Copy to resolve this error "[ Use of moved value.](https://github.com/ChainSafe/mina-rs/pull/80/commits/68772460e890148d6d1630eac2248255bb480ac2)"